docs/releases/5.2.txt

```diff
@@ -253,7 +253,8 @@ Signals
 Templates
 ~~~~~~~~~
 
-* ...
+* The :func:`~django.template.context_processors.debug` context processor
+  is removed from the list in the default generated settings.
```

Comment:
As far as I'm aware, this is a breaking change not a new feature so it should go to the `Backwards incompatible changes in 5.2` section.

Reply:
I think we can be a little clearer:

```suggestion
  is removed from default project template.
```

I guess it's very slightly backwards incompatible, in the case someone has scripted generating Django projects and relies on the output containing the debug tag...

Reply:
> I think we can be a little clearer:

@adamchainz thank you for better words!

> As far as I'm aware, this is a breaking change

@felixxm I had considered putting it there as well but decided against it given that the change is in `startproject` and shouldn't affect existing projects being upgraded. Thoughts?

Reply:
I have slightly tweaked this message and I'll move it outside the "minor features" section since, while I agree is tangentially backwards incompatible, it is certainly not a new feature.